samples/excel/50-workbook/create-workbook.yaml

```diff
@@ -0,0 +1,94 @@
+order: 2
+id: excel-workbook-create-workbook
+name: Create workbook
+description: Creates new workbooks both from scratch and by copying an existing one.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.8
+script:
+    content: |
+        $("#create-new-blank-workbook").click(() => tryCatch(createBlankWorkbook));
+        $("#file").change(() => tryCatch(createWorkbookFromExisting));
+
+        async function createBlankWorkbook() {
+            await Excel.run(async (context) => {
+                Excel.createWorkbook();
+            });
+        }
+
+        async function createWorkbookFromExisting() {
+            const myFile = document.getElementById("file");
+            const reader = new FileReader();
+
+            reader.onload = ((event) => {
+                Excel.run(context => {
+                    // strip off the metadata before the base64-encoded string
+                    const startIndex = event.target.result.indexOf("base64,");
+                    const mybase64 = event.target.result.substr(startIndex + 7);
+
+                    Excel.createWorkbook(mybase64);
+                    return context.sync();
+                });
+            });
+
+            // read in the file as a data URL so we can parse the base64-encoded string
+            reader.readAsDataURL(myFile.files[0]);
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to create a new workbook by copying an existing one.</p>
```

Comment:
Seems like, based on additional scenario, this paragraph should be updated

Reply:
+1 Perhaps something like -> "This sample shows how to create a new (empty) workbook and how to create a new workbook by copying an existing one."